quickstarts/angular1/01-login.md

```diff
@@ -0,0 +1,222 @@
+
+Angular's `config()` skeleton with required dependencies:
+```javascript
+app.config( function myAppConfig (authProvider) {
+  //authProvider init configuration
+})
+```
+
+The `authProvider` dependency is Auth0's API that exposes some methods including the `init()` method for for configuration:
+
+${snippet(meta.snippets.init)}
+
+Event listeners are available to handle different status of authentication. They are also configured in the `config()` method with `authProvider`:
+
+```javascript
+//Called when login is successful
+authProvider.on('loginSuccess', function(idToken) {
+  console.log("Login successful with token " + idToken);
+});
+
+//Called when login fails
+authProvider.on('loginFailure', function() {
+  alert("Error");
+});
+
+//Called when user is authenticated (Authentication flag is true)
+authProvider.on('authenticated', function() {
+  console.log("Authenticated");
+});
+```
+
+#### 3: Login
+
+You have successfully configured you Angular App to use Auth0. Configs are vital but won't run an app, right? Let's add a login functionality:
+
+```javascript
+app.controller( 'LoginCtrl', function ( $scope, auth) {
+  // Add auth to $scope object so we can bind to view
+  $scope.auth = auth;
+
+});
+```
+`auth` is one of the goodies (service) you get for including `auth0` as one of the dependencies. Passing `auth` to `$scope.auth` gives you the power to bind `auth` data and events to the view:
+
+```html
+<!-- Inside index.html <body> -->
+<div ng-controller="LoginCtrl">
+  <a href="#" class="btn btn-default" ng-click="auth.signin()">Sign In</a>
+</div>
+```
+
+At this point, the lock widget will pop up showing a Sign In form,  when you click the `Sign In` button.
+
+${browser}
+
+You can change this behavior and make it show the Sign Up form:
+
+```html
+<a href="#" class="btn btn-default" ng-click="auth.signup()">Sign Up</a>
+```
+
+The event listeners that we created in our app's config will be called to handle the sign in process.
+
+Moving the `singin()` logic into your controller rather than leaving it in the view will give you more manipulation power:
+
+```javaScript
+//app.js
+$scope.signin = function (){
+  auth.signin({}, //The first argument is scope of data you need to return
+    function(profile, idToken){
+      $scope.token = idToken;
+    },
+    function(err) {
+      $scope.err = err;
+    });
+}
+```
+Now you have access to the profile and token:
+
+```html
+<!-- Show when not signed in -->
+<a ng-if="!token" class="btn btn-success btn-lg" ng-click="signin()">Sign In</a>
+<!-- Show token when signed in -->
+<p ng-if="token">{{token}}</p>
+```
+
+### Recap and Round Up
+At this point, you have learned how to:
+- Setup Auth0 in an Angular project
+- Configure Angular with Auth0 credentials and authentication status events
+- Login
+
+As mentioned earlier, this is just the beginning of a multi-step quickstart. You can continue to the next step when you are done here.
+
+::: panel-info Next Step
```

Comment:
Remove this part, we will generate the links to each step automatically for each document.